library/pkwrite.h

```diff
@@ -105,4 +105,27 @@
 
 #endif /* MBEDTLS_ECP_C */
 
+#if defined(MBEDTLS_ECP_LIGHT)
```

Comment:
Note: this would be better placed in `library/pk_internal.h`. This file does not exist yet, but it's going to be created soon, see #7202. So, in order to avoid conflicts, let's keep this here for the time being, and then @valeriosetti can move it to the internal header when in his PR that creates it.